### PR TITLE
R translation bain

### PR DESCRIPTION
**Pull request by JorisGoosen**
I'm still working on 'em, just wanna see what Travis has to say about it for now.

### CODE REVIEW

**Comment by JorisGoosen** [comment]
I've done them now.

@koenderks it seems quite some resultstable functions are copied and changed very little. At least in the ttests for sure. Could these functions not be merged?

**Comment by koenderks** [comment]
Yes, that can probably happen. I've been focused on https://github.com/jasp-stats/jasp-desktop/pull/3707 and https://github.com/jasp-stats/jasp-desktop/pull/3716, but may be able to sqeeze some time in for bain before the release. Can you make an issue amd assign me?

**Comment by JorisGoosen** [comment]
I've made an issue for you, if you think you can finish it you can just move it to 'medium priority' in the backlog project.

File: JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R [hunk 1]
```diff
@@ -226,51 +226,51 @@ BainTTestBayesianIndependentSamples <- function(jaspResults, dataset, options, .
...
-                               "type[equal]" = "H0: Equal",
+                               "type[equal]" = gettext("H0: Equal"),
                                "BF[equal]"= "",
                                "pmp[equal]" = PMP_0,
-                               "type[greater]" = "H1: Bigger",
+                               "type[greater]" = gettext("H1: Bigger"),
                                "BF[greater]" = BF_01,
                                "pmp[greater]" = PMP_1,
-                               "type[less]" = "H2: Smaller",
+                               "type[less]" = ("H2: Smaller"),
```
Comment (FransMeerhoff) [inline JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R]: gettext("H2: Smaller")

**Pull request merged**